app/views/parents/student-progress/CocoModuleProgressComponent.vue

```diff
@@ -77,7 +77,7 @@ export default {
   },
   methods: {
     getIconType (level) {
-      if (level.kind === 'practice' || level.practice) return 'practicelvl'
+      if (level.kind === 'practice' || level.practice !== false) return 'practicelvl'
```

Comment:
https://github.com/codecombat/codecombat/blob/b16891cb3b60b6c4d17d2454a419e390c2ed07eb/app/core/utils.js#L1603-L1615 you can follow this logic to check practice or challenge. note that all `assessment` lead to challenge.

Reply:
and if a level not match in practice or challenge we can also show as `challenge` icon

Reply:
updated! level.practice always seems to return false or undefined so level.kind is used instead. I'm curious, what even broke the code in the first place?